image_enforcer/README.md

```diff
@@ -0,0 +1,184 @@
+# Image Enforcer
+
+This directory shows how you can use OPA to enforce admission policies on
+container images in Kubernetes.
+
+You can use this project to enforce simple admission policies on images such as:
+
+- Naming conventions
+- Version pinning (e.g., no latest)
+- Registry usage (e.g., production must use internal registry)
+
+Using the `openpolicyagent/clair-layer-sync` image, you can also enforce more
+sophisticated policies based on image vulnerability scanning.
+
+This directory contains the following:
+
+- Kubernetes manifests for deploying the image enforcer.
+- Steps to use the image enforcer.
+- Source for the `openpolicyagent/clair-layer-sync` Docker image.
+
+## Testing
+
+This repository contains sample policies and a data set that includes layer metadata with known vulnerabilities.
+
+### Prerequisites
+
+- OPA v0.4.10 or later
```

Comment:
Also can this run on minikube? If not, would be nice to see a brief explanation of why not